tests/test_nc_to_mmd.py

```diff
@@ -1395,7 +1428,7 @@ def test_missing_platform_vocabulary(self):
         ncin = Dataset(md.netcdf_file, "w", diskless=True)
         ncin.platform = 'Envisat'
```

Comment:
ok. The real long name is environmental satellite, although it doesn't really matter here.